qpmodel/Program.cs

```diff
@@ -145,19 +146,19 @@ static void Main(string[] args)
                 Tpch.LoadTables("0001");
                 //Tpch.CreateIndexes();
                 Tpch.AnalyzeTables();
-                sql = File.ReadAllText("../../../tpch/q20.sql");
+                sql = File.ReadAllText("../../../../tpch/q20.sql");
                 goto doit;
             }
 
             if (false)
-            { 
+            {
                 Tpcds.CreateTables();
                 Tpcds.LoadTables("tiny");
                 Tpcds.AnalyzeTables();
                 // 1, 2,3,7,10,
                 // long time: 4 bad plan
                 // 6: distinct not supported, causing wrong result
-                sql = File.ReadAllText("../../../tpcds/q7.sql");
+                sql = File.ReadAllText("../../../../tpcds/q7.sql");
```

Comment:
The file paths changes due to dotnet core

Reply:
ok